styles/globals.css

```diff
@@ -4,29 +4,16 @@
 
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&family=Raleway:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,200;1,300&display=swap");
 
-:root {
+/* :root {
```

Comment:
Why this line?

Reply:
Ig it's for all elements correct me if i am wrong